### ISSUE TITLE
[OS X] client uses hardcoded user settings path

### ISSUE THREAD

**Issue by ponychicken** [owncloud/client#4981]
OS X 10.11.5, OwnCloud 2.1.1
1. Setup a User account with a non standard user folder location (e.g. /Volumes/ExternalDisk/Users)
2. Install OwnCloud
3. Observe that OwnCloud will set it's folder up in /Volumes/StartUpDisk/Users

**Comment by guruz** [comment]
I don't think this is something we're going to change. What is the reason for your report?

**Comment by ponychicken** [comment]
Because all Apps that I used respected the custom path setting, while Owncloud went and recreated the old user folder..
